fuzzing/fuzzer_test.go

```diff
@@ -486,13 +486,16 @@ func TestDeploymentsWithPredeploy(t *testing.T) {
 func TestDeploymentsWithPayableConstructors(t *testing.T) {
 	runFuzzerTest(t, &fuzzerSolcFileTest{
 		filePath: "testdata/contracts/deployments/deploy_payable_constructors.sol",
-		configUpdates: func(config *config.ProjectConfig) {
-			config.Fuzzing.TargetContracts = []string{"FirstContract", "SecondContract"}
-			config.Fuzzing.TargetContractsBalances = []*big.Int{big.NewInt(0), big.NewInt(1e18)}
-			config.Fuzzing.TestLimit = 1 // this should happen immediately
-			config.Fuzzing.Testing.AssertionTesting.Enabled = false
-			config.Fuzzing.Testing.OptimizationTesting.Enabled = false
-			config.Slither.UseSlither = false
+		configUpdates: func(pkgConfig *config.ProjectConfig) {
+			pkgConfig.Fuzzing.TargetContracts = []string{"FirstContract", "SecondContract"}
+			pkgConfig.Fuzzing.TargetContractsBalances = []config.ContractBalance{
+				{Int: *big.NewInt(0)},
+				{Int: *big.NewInt(1e18)},
+			}
```

Comment:
I think this is great. Let's add one more for a hex string? So update the fuzz test ("testdata/contracts/deployments/deploy_payable_constructors.sol") to have a `ThirdContract` and send it funds using a hex string.